### PR TITLE
Test rubycritic

### PR DESCRIPTION
**Pull request by ajmpawlik**
We added rubycritic as gem. Remember to do bundle install. The command is rubycritic app app/controllers/application_controller.rb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ Navigate to the root of the project then run the tests from the command line
 admin$ rspec
 ```
 
+## How to analyse the complexity of the commits
+Navigate to the root of the project then run the tests from the command line
+
+```console
+admin$ rubycritic app app/controllers/application_controller.rb
+```
```
Comment (rachjgriff) [inline README.md]: might be nice to put an example of the output from ruby critic so we know what it does? (coming from someone who has no idea!!!)

Comment (ajmpawlik) [reply README.md]: I love the idea. Will do it now!

Comment (Danielandro) [reply README.md]: When would we need to run this command? Before/after every commit? Pull request?

Comment (ajmpawlik) [reply README.md]: Done.

**Pull request merged**